response_operations_ui/views/collection_exercise.py

```diff
@@ -307,19 +327,29 @@ def _validate_collection_instrument():
             logger.info('Invalid file format uploaded', filename=file.filename)
             error = {
                 "section": "ciFile",
-                "header": "Error: wrong file type for collection instrument",
+                "header": "Error: Wrong file type for collection instrument",
                 "message": "Please use XLSX file only"
             }
         else:
-            # file name format is surveyId_period_formType
-            form_type = _get_form_type(file.filename) if file.filename.count('_') == 2 else ''
-            if not form_type.isdigit() or len(form_type) != 4:
-                logger.info('Invalid file format uploaded', filename=file.filename)
-                error = {
-                    "section": "ciFile",
-                    "header": "Error: invalid file name format for collection instrument",
-                    "message": "Please provide file with correct form type in file name"
-                }
+            if file.filename.split(".")[0].isdigit():
+                logger.info("BRES collection instrument detected", filename=file.filename)
+                if len(file.filename.split(".")[0]) != 11:
+                    logger.info('Invalid BRES file format uploaded', filename=file.filename)
+                    error = {
+                        "section": "ciFile",
+                        "header": "Error: Invalid file name format for BRES collection instrument",
+                        "message": "Please provide a file with a valid 11 digit ru ref in the file name"
+                    }
```

Comment:
All these nested ifs feel a little complicated to me, I think we should split this function. See above